system-tests/tests/load/cre/workflow-don-load-test-streams-crib.toml

```diff
@@ -7,11 +7,6 @@ chain_id = "1337"
 type = "anvil"
 docker_cmd_params = ["-b", "5"]
 
-# ----- BINARY PATHS -----
-# Path to the mock capability binary
-[binaries_config]
-mock_capability_binary_path = "../../../../../capabilities/bin/amd64/mock"
```

Comment:
don't we need in case someone wants to use locally built mock capability with some existing image?

Reply:
Yes but I would say that this is outside the scope of these tests